include/OsqpEigen/Settings.hpp

```diff
@@ -89,7 +89,13 @@ namespace OsqpEigen
          * Set the max number of iterations.
          * @param maxIteration max number of iteration
          */
-        void setMaxIteraction(const int maxIteration);
+        [[deprecated("Use setMaxIteration(int) instead.")]] void setMaxIteraction(const int maxIteration);
+
+        /**
+         * Set the max number of iterations.
+         * @param maxIteration max number of iteration
+         */
+		void setMaxIteration(const int maxIteration);
```

Comment:
Thanks @kalj ! Can you keep using spaces instead of tabs, as done in the rest of the file?

Reply:
Oh, crap. Absolutely.